src/client/containers/CandidateList/CandidateList.js

```diff
@@ -25,7 +25,7 @@ export default function CandidateList({ userId, boardId }) {
     if (firstCandidates !== candidates) {
       updateBallots(userId, boardId, candidates);
     }
-  }, [candidates]);
+  }, [userId, boardId, candidates]);
```

Comment:
Lint error triggers if I don't use userId and boardId in dependency.

Reply:
I was thinking :
  const [draggedInit, setDraggedInit] = useState(false);
  
  then pass the seDraggedInit to the onDragEnd function and set it to true from there

Reply:
And also:
I think we could actually bring back the useEffect back in updateBallots to make it more React-like, as you initially suggested.

```
import { useEffect } from 'react';

export async function updateBallots(userId, boardId, candidates, draggedInit) {

  useEffect(() => {
    if (!draggedInit) return;
    ;(async () => {
      await fetch(
        `/api/users/${userId}/boards/${boardId}/ballots`,
        {
          method: 'PUT',
          mode: 'cors',
          headers: {
            'Content-type': 'application/json',
          },
          body: JSON.stringify(candidates),
        },
      );
    })();  
  }, [candidates])
}
```
maybe change the name to useUpdateBallots to keep with the convention

Reply:
It's not sending put request. Where am I doing wrong?